assets/schemas/manifest.json

```diff
@@ -1333,9 +1333,6 @@
           "description": {
             "type": "string"
           },
-          "tcfId": {
```

Comment:
Why removed?

Reply:
may be able to remove based on purpose code logic of tcf.1 but will leave in pipeline for options